src/app/project/compliance/inspection-detail/inspection-detail-resolver.service.ts

```diff
@@ -12,6 +12,6 @@ export class InspectionDetailResolver implements Resolve<Observable<object>> {
 
   resolve(route: ActivatedRouteSnapshot): Observable<object> {
     const inspectionId = route.paramMap.get('inspectionId');
-    return this.searchService.getItem(inspectionId, 'Inspection');
+    return this.searchService.getItem(inspectionId, 'Inspection', true);
```

Comment:
For me, when this is true, viewing inspections always fails. If I set it to false (disabling the cache), viewing inspections works fine.